docs/glossary.md

```diff
@@ -0,0 +1,86 @@
+
+- **Example (Monthly):** If $P_i$ is 0.1 and you have 100 months of observation, $E^m_i$ would be 10 months, meaning the event is expected to occur once every 10 months.
+
+## 9. **At Least One Occurrence Probability (P_i)**
+- **Definition:** The probability of observing at least one event of a specific size or larger across multiple grid cells within a specific time period. If $p_i$ is the event probability and $\text{num\_grid\_cells}$ is the total number of grid cells, we have:
+
+- **Formula:** $$P_i = 1 - (1 - p_i)^{\text{num\_grid\_cells}}$$
+
+- **Example:** If $p_i$ is 0.1 and you have 100 grid cells, $P_i$ represents the probability of seeing at least one event of the specified size in any of the 100 grid cells.
+
+## 10. **Expected Annual Losses (EAL)**
+- **Definition:** The average annual financial loss expected due to events of a certain size or larger. It integrates the probability of events with their associated costs.
+
+- **Example:** If the average annual loss due to conflicts in a region is calculated to be $1 million, this would represent the EAL.
+
+## 11. **Exceedance Probability (EP)**
+- **Definition:** The probability that a specific event's magnitude will be exceeded within a given time period. It is often used in risk management to understand the likelihood of extreme events.
+
+- **Example:** An exceedance probability of 0.05 for 100 fatalities per 100k population means there is a 5% chance of observing more than 100 fatalities per 100k population in a given time period.
+
+## 12. **Parametric Insurance**
+- **Definition:** A type of insurance that pays out when predefined conditions (parameters) are met, such as when an event of a certain magnitude occurs, rather than compensating for actual losses incurred.
+
+- **Example:** A parametric insurance policy might pay out if the number of fatalities per 100k population exceeds a certain threshold within a given time period.
+
+## 13. **Temporal Aggregation**
+- **Definition:** The process of summing or averaging data over time to form a single time period. Temporal aggregation is used to create annual, monthly, or other time-based summaries from finer-grained data.
+
+- **Example:** Aggregating daily fatalities per 100k population data to produce a monthly total.
+
+## 14. **Spatial Aggregation**
+- **Definition:** The process of combining data from multiple spatial units (e.g., grid cells) to form a larger spatial unit for analysis.
+
+- **Example:** Aggregating fatalities data from individual grid cells to generate a country-level summary.
+
+## 15. **Empirical Probability**
+- **Definition:** The probability derived from observed data, as opposed to theoretical probability. In this context, it refers to the likelihood calculated from actual events in the dataset.
+
+- **Example:** If 10 out of 100 observed grid cells have more than 50 fatalities per 100k population, the empirical probability of this event is 0.1.
```

Comment:
Had to read this example twice to understand that _"this event"_ referred to having more than 50 fatalities per 100k. Consider: _"... the empirical probability of observing 50 fatalities per 100k population within a given grid cell is 0.1"_. Unless if this may also make it confusing for the time-based event return period.